acai-ios/DetailViewControllers/SmoothieOrderViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  SmoothieOrderViewController.swift
```

Comment:
Is the smoothie different? We should be laying out the smoothies and bowls and coffees in the same views dynamically with the data.

Reply:
I planning to make one detailView for smoothies, drinks, and bowls, but they're still separate bc they're going to have a different number of customization section controllers and having an array for [OrderCustomizationOptions] in the detail view wasn't working with updating the isSelected.

Reply:
We should be able to create the different number of section controllers and such dynamically for each type, and yeah we'll need to pack that array as another ListDiffable model to represent the different amounts of options